### PR TITLE
TWI-44 Create env file

### PR DESCRIPTION
**Pull request by pm888**

### CODE REVIEW

File: cmd/app/main.go [hunk 1]
```diff
@@ -1,22 +1,44 @@
...
 
 type ServiceMongoDb struct {
 	DB interface{}
 }
 
 func main() {
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go func() {
+		err := admin.LoadEnvFile()
```
Comment (bunin) [inline cmd/app/main.go]: where is this code?

**Review by bunin — CHANGES_REQUESTED** [review]

File: cmd/app/main.go (hunk 1 above)
Comment (pm888) [reply cmd/app/main.go]: done

File: internal/database/pg/pg.go [hunk 2]
```diff
@@ -24,11 +24,12 @@ func ConnectPostgresql() error {
 
 	db, err := sql.Open("postgres", connStr)
```
Comment (bunin) [inline internal/database/pg/pg.go]: you are still connecting using hardcoded values

File: .env [hunk 3]
```diff
@@ -0,0 +1,7 @@
+DB_PG_HOST=localhost
+DB_PG_PORT=55000
+DB_PG_USER=postgres
+DB_PG_PASSWORD=postgrespw
```
Comment (bunin) [inline .env]: you should not push secrets

**Review by bunin — CHANGES_REQUESTED** [review]

File: cmd/app/main.go [hunk 4]
```diff
@@ -1,22 +1,44 @@
...
+		if err != nil {
+			fmt.Println(err)
+		} else {
+			fmt.Println("**** env files is completed ****")
+		}
+		wg.Done()
+	}()
+	wg.Wait()
```
Comment (bunin) [inline cmd/app/main.go]: this waitgroup makes no sense

File: internal/server/server.go [hunk 5]
```diff
@@ -12,7 +13,7 @@ import (
 
 func Server() {
 	r := mux.NewRouter()
-	fmt.Println("Server was run", "localhost:8080")
+	fmt.Println(fmt.Sprintf("Server was run %s:%s", admin.ServerHost, admin.ServerPort))
```
Comment (bunin) [inline internal/server/server.go]: you've changed the log output but not the actual listen address

**Review by bunin — CHANGES_REQUESTED** [review]

File: internal/server/server.go [hunk 6]
```diff
@@ -10,9 +11,9 @@ import (
 	"net/http/httptest"
 )
 
-func Server() {
+func Server() error {
 	r := mux.NewRouter()
-	fmt.Println("Server was run", "localhost:8080")
+	fmt.Println(fmt.Sprintf("Server was run %s:%s", admin.ServerHost, admin.ServerPort))
```
Comment (bunin) [inline internal/server/server.go]: there's fmt.Printf

**Review by bunin — CHANGES_REQUESTED** [review]
https://github.com/mo-pm888/twitter/pull/34/files#r1255801443

**Review by bunin — APPROVED** [review]

**Pull request merged**